exchange/utils_test.go

```diff
@@ -91,6 +92,24 @@ func TestCleanOpenRTBRequestsCCPA(t *testing.T) {
 			enforceCCPA:     true,
 			expectDataScrub: true,
 		},
+		{
+			description:     "Feature Flag Enabled - No Sale Star - Doesn't Scrub",
+			reqExt:          json.RawMessage(`{"prebid":{"nosale":["*"]}}`),
+			enforceCCPA:     true,
+			expectDataScrub: false,
+		},
+		{
+			description:     "Feature Flag Enabled - No Sale Specific Bidder - Doesn't Scrub",
+			reqExt:          json.RawMessage(`{"prebid":{"nosale":["appnexus"]}}`),
+			enforceCCPA:     true,
+			expectDataScrub: false,
+		},
+		{
+			description:     "Feature Flag Enabled - No Sale Different Bidder - Scrubs",
+			reqExt:          json.RawMessage(`{"prebid":{"nosale":["rubicon"]}}`),
+			enforceCCPA:     true,
+			expectDataScrub: true,
+		},
```

Comment:
The tests in this part of code are a bit expensive since there's a lot of logic to flow through. My intention was to provide a basic "does this work" test here for sanity. The `TestShouldEnforce` in `policy_test.go` covers edge cases.

Reply:
I added a test case there.